project_milestone_status/models/__init__.py

```
@@ -0,0 +1,2 @@
+from . import project_milestone
```

Ruff (0.8.2)

`.project_milestone` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)